### ISSUE TITLE
Can't hook URLSearchParams.get

### ISSUE THREAD

**Issue by kiredevsandhacks** [kevin-mizu/domloggerpp#21]
I seem to unable to hook into URLSearchParams, using this config:
```
{
  "config": {},
  "hooks": {
    "URL params": {
      "function": [
        "URLSearchParams.prototype.get",
      ]
    }
  }
}
```

When manually pasting `(new URLSearchParams(location.search)).get('test')` in the console, I still get no result. I also tried `URLSearchParams.get` or a custom hook `function:URLSearchParams.get`, `function:URLSearchParams.prototype.get`,  but all to no avail.

Or is URLSearchParams a special type that can't be hooked like this?

**Comment by kevin-mizu** [comment]
Hi 👋

Thank you for reporting this! However, it has already been reported and fixed by @matanber  in PR #15. The fix should be included in the next release :D

**Comment by kiredevsandhacks** [comment]
Ahhh I missed that one. Thanks! I checked to see if this issue was already present/resolved but I looked over it. Will close this one.
